$ OpenBSD: $

Forcing linking with opus from /usr/local instead of bundled.
XXX: check the relevance

Index: Telegram/CMakeLists.txt
--- Telegram/CMakeLists.txt.orig
+++ Telegram/CMakeLists.txt
@@ -4,6 +4,7 @@
 # For license and copyright information please follow this link:
 # https://github.com/telegramdesktop/tdesktop/blob/master/LEGAL
 
+link_directories(${OPUS_LIBDIR})
 add_executable(Telegram WIN32 MACOSX_BUNDLE)
 init_non_host_target(Telegram)
 
